vlasovsolver/vlasovmover.cpp

```diff
@@ -272,9 +256,11 @@ void calculateSpatialTranslation(
    
    // If dt=0 we are either initializing or distribution functions are not translated. 
    // In both cases go to the end of this function and calculate the moments.
-   if (dt == 0.0) goto momentCalculation;
+   if (dt == 0.0) {
+      calculateMoments_R(mpiGrid,localCells,true);
+   }
```

Comment:
I would add a return here so we don't do a pointless zero-dt translation computation. Then it'll return to the same behaviour we had before.

Reply:
That was the intention, I think I goofed up the merge there